webclient/pas/views.py

```diff
@@ -132,6 +132,10 @@ def _set_copy_source(self):
         copy_source.id = None
         copy_source.soubory = None
         copy_source.historie = None
+        copy_source.evidencni_cislo = None
+        copy_source.predano_organizace = None
+        copy_source.predano = None
+        copy_source.pristupnost = None
```

Comment:
Toto má výchozí hodnotu "archeolog", ale také předpokládám, že se to řeší jinde.